app/controllers/v1/projects_controller.rb

```diff
@@ -76,6 +76,30 @@ def destroy
       end
     end
 
+    swagger_api :list_tasks do
+      summary "List all of a project's tasks"
+      notes "This lists all of the selected project's the active tasks"
+      param :id
+    end
+    def list_tasks
+      project = Project.find_by(id: params[:id])
+      tasks = project.tasks
+      render json: tasks, each_serializer: V1::TaskSerializer
+    end
+
+    swagger_api :create do
+      summary 'Creates a new Task for the selected Project'
+      param :name, :project_id
+    end
+    def create_task
+      task = Task.new(name: params[:name], project_id: params[:project_id])
```

Comment:
I didn't want to force the user to have to pass a description because I thought it would be a better user experience to have it be optional. For example, I use an app to organize work tasks called Wunderlist. It's similar to this project in the fact that it has projects and tasks. Usually, I like to create a project and then create tasks underneath the project with names that are self-explanatory. If I have details or notes about the project that I would find useful, I add them to my project's description notes. I thought that as a user, when I create a list of tasks, the tasks tend to be self-explanatory. So my thinking was that it might grow tedious if every time I created a task, I was forced to also give it a description.

This also could be a nooby thing, but I was under the impression that when you don't implicitly set a param as an argument(like I did above) that it would let the parameter be optional? I'd love to see a better way to handle this.